CMakeGlobals.txt

```diff
@@ -200,8 +203,20 @@ if(NOT DEFINED RSTUDIO_VERIFY_R_VERSION)
    endif()
 endif()
 
+# pandoc version
 set(PANDOC_VERSION "2.16.1" CACHE INTERNAL "Pandoc version")
```

Comment:
Let's update this to Pandoc 2.16.2

Reply:
Note we'd need to update the pandoc dependency installation script as well as this: https://github.com/rstudio/rstudio/blob/main/dependencies/tools/upload-pandoc.sh